### ISSUE TITLE
Seq Lab: New Python class for Firestore QC data

### ISSUE THREAD

**Issue by tim-snyder-wustl** [GTAC-MGI/GTAC-ESP-LIMS#237]
Blocked by: https://github.com/GTAC-MGI/GTAC-ESP-LIMS/issues/236
Relates to: https://github.com/GTAC-MGI/GTAC-ESP-LIMS/issues/222

---

We need from Mark:

Filepaths for FASTQ files, and which sample they belong to
The actual QC stats
SR# for samples
Show these Library QC Metrics:

https://wustl.app.box.com/file/950842021254

Example:

https://app.smartsheet.com/sheets/J7g3JgmFw7QFQmcFHqxqJFPF9p38rc7mg6GgJ541?view=grid

There should be combined totals for libraries that may have been run over multiple lanes as well.

Need

- SR # associated with QC metrics
- Sample Name / ID

[dragen_Reports.tar.gz](https://github.com/GTAC-MGI/GTAC-ESP-LIMS/files/11354526/dragen_Reports.tar.gz)

**Comment by tim-snyder-wustl** [comment]
No longer needed, closing.
